.github/workflows/chromatic.yml

```
@@ -0,0 +1,72 @@
+name: 'Chromatic Auto Deploy on PR to Main'
+
+on:
+  pull_request:
+    types: [labeled]
+    branches:
+      - '**' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)